nupic/research/frameworks/vernon/trainables.py

```diff
@@ -64,7 +76,13 @@ def default_resource_request(cls, config):
         return resource
 
     def _setup(self, config):
-        self.experiment_class = config["experiment_class"]
+        experiment_class = config["experiment_class"]
+
+        class ExtendedExpClass(RayActorHelperMethods, experiment_class):
+            pass
+
+        ExtendedExpClass.__name__ = experiment_class.__name__
+        self.experiment_class = ExtendedExpClass
```

Comment:
This could be replaced by `insert_experiment_mixin` used by the other mixins like the profiler option

Reply:
`insert_experiment_mixin` modifies the `config` in place. I don't want to do that here.